test/contrib/automotive/uds.uts

```diff
@@ -1423,3 +1423,10 @@ nrc = UDS(b'\x7f\x22\x33')
 assert nrc.service == 0x7f
 assert nrc.requestServiceId == 0x22
 assert nrc.negativeResponseCode == 0x33
+
+
+= Check UDS get negative Response Description
+
+nrcDescription = UDS_NR.negativeResponseCodes.get(0x50)
+assert nrcDescription == 'Certificate verification failed - Invalid Time Period'
```

Comment:
this will never work! You need to change the string to: "certificateVerificationFailedInvalidTimePeriod"

However, this code tests the "get" of a "dict" object. I doubt that this test case is useful.

Reply:
Okay lets remove the test then